htdocs/index.php

```diff
@@ -18,6 +18,12 @@
 require_once("../lib/functions.inc.php");
 if ($use_captcha) {
     require_once("../lib/captcha.inc.php");
+} else if ($change_apppwd != false) {
+    for ($i = 0; $i < count($change_apppwd); $i++) {
+        if (isset($change_apppwd['use_captcha']) && $change_apppwd['use_captcha']) {
```

Comment:
of course it has to be 
`if (isset($change_apppwd[i]['use_captcha']) && $change_apppwd[i]['use_captcha']) {`
will fix today...